src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,86 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+
+import { Person } from '../../types';
+import { PeopleLink } from '../PeopleLink';
+import { findParent } from '../../helpers/helpers';
+
+export type Props = {
+  people: Person[],
+  personId: string,
+};
+
+export const PeopleTable: FC<Props> = (props) => {
+  const {
+    people,
+    personId,
+  } = props;
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map((person) => {
+          const {
+            sex,
+            born,
+            died,
+            motherName,
+            fatherName,
+            slug,
+          } = person;
+
+          const motherOfPerson = findParent(people, motherName);
+          const fatherOfPerson = findParent(people, fatherName);
```

Comment:
Let's prepare array to working with it and save needed fields there instead of calling `findParent` on every render